Formula/kumactl.rb

```diff
@@ -0,0 +1,26 @@
+class Kumactl < Formula
+  desc "Kuma control plane command-line utility"
+  homepage "https://kuma.io/"
+  url "https://github.com/Kong/kuma.git",
+      :tag      => "0.5.0-rc2",
```

Comment:
https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
since this is a release cut and it is a stable version, should i need to downgrade to a lower version?

Reply:
The repo claims the latest release is 0.4.0 so you should submit that one or wait until 0.5.0 becomes stable.

Reply:
okay